File: Flights/ReadModels/BookingRm.cs
namespace FlightsSearchPortal.ViewModels
{
    public record BookingRm(
        int FlightId,
        string Airline,
        string Price,
        TimePlaceRm Arrival,
        TimePlaceRm Departure,
        int NumberOfSeats,
        string PassengerEmail);
}

File: flight-booking-portal/src/app/api/models/booking-rm.ts
/* tslint:disable */
/* eslint-disable */
import { TimePlaceRm } from '../models/time-place-rm';
export interface BookingRm {
  airline?: string | null;
  arrival?: TimePlaceRm;
  departure?: TimePlaceRm;
  flightId?: number;
  numberOfSeats?: number;
  passengerEmail?: string | null;
  price?: string | null;
}
